ex15/main.go

```diff
@@ -1,5 +1,23 @@
 package main
 
-  func main() {
+var justString string
 
-  }
+func someFunc() {
+	v := createHugeString(1 << 10)
+	//justString = string(v[:100])
+	tmp := make([]byte, 0, 100)
+	copy(tmp, v)
+	justString = string(tmp)
+
+}
+
+func createHugeString(size int) []byte {
+	return make([]byte, 0, size)
+}
```

Comment:
In the task, we talk about the `string` type not `[]byte`
I'm not sure about the right solution because if we use `make()` it will always be allocated on the heap
https://go101.org/optimizations/0.3-memory-allocations.html
https://www.sobyte.net/post/2022-03/go-escape/